wikimedia/wikipedia-ios
# Add subscription methods to TalkPageFetcher

**Phabricator:** https://phabricator.wikimedia.org/T311071 

### Notes
* This PR adds methods for subscribing/unsubscribing to topics and getting all the topics currently subscribed

### Test Steps
1. Run the app on staging
2. Add a call to the methods to our new [TalkPageViewController](https://github.com/wikimedia/wikipedia-ios/blob/4b927a509a147ec02ae323bf728746620a818089/Wikipedia/Code/TalkPageViewController.swift#L31)
3. Run and check on the web if it was successful (subscribing/unsubscribing and getting all the current subscribed topics)
